### ISSUE TITLE
panelOpen: true / false should be a init config so that this can be used with bloc 

### ISSUE THREAD

**Issue by 5hanth** [akshathjain/sliding_up_panel#82]
if such option is present, panel can be used with immutable states and easy to re-render panel with a different state.  currently panel controller cant be stored in  immutable state.

**Comment by rohankandwal** [comment]
Yes, currently trying to open the drawer on a button click, it isn't working.

**Comment by omidraha** [comment]
Exactly I have same issue.
